website/frontend/src/pages/CareerPage.js

```diff
@@ -101,7 +101,7 @@ const CareerPage = () => {
 
                         <div className="self-intro">
                             <span>Don’t see a <br/>position that fits you perfectly? Introduce yourself here </span>
-                            <span>careers@airqo.africa</span>
```

Comment:
I am not sure if this will work. I am only sure about careers@airqo.net.

Do our emails work on .africa? @FaithDaka

Reply:
That is the change being made @Baalmart. The red line means it was removed, so in green you'll see 'careers@airqo.net'.
Thanks!

Reply:
Thanks @FaithDaka